internal/server/api.yaml

```diff
@@ -283,6 +304,46 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/UploadRequest'
+    ComposesResponse:
+      required:
```

Comment:
Where are these documented? I assume they are standard?

Reply:
Hm what do you mean?

Reply:
What are the `meta` and `links` properties? How are they meant to be used? Why did you add them? I assumed they were some standard thing, in which case, where can I read more about them?